pyrs/peaks/peak_collection.py

```diff
@@ -155,6 +156,43 @@ def set_peak_fitting_values(self, subruns, parameter_values, parameter_errors, f
         self._fit_cost_array = np.copy(fit_costs)
         self.__set_fit_status()
 
+    def get_d_reference(self):
+        return self._d_reference
+
+    def set_d_reference(self, values):
+        """Set d reference values
+
+        Parameters
+        ----------
+        values :
+            1D numpy array or floats
+
+        Returns
+        -------
+
+        """
+        self._d_reference = [values] * self._sub_run_array.size
+
+    def get_strain(self, values=np.nan):
+        """get strain values and uncertainties
+
+          Parameters
+          ----------
+          values :
+              1D numpy array or floats
+
+          Returns
+          -------
+            tuple
+                A two-item tuple containing the strain and its uncertainty.
+          """
+        d_fitted, d_fitted_error = self.get_dspacing_center()
+        self.set_d_reference(values)
+        strain = (d_fitted - self.get_d_reference())/self.get_d_reference()
```

Comment:
No need to go through the public functions to access instance values
```suggestion
        strain = (d_fitted - self._d_reference)/self._d_reference
```